lib/Staticman.js

```diff
@@ -46,7 +46,7 @@ Staticman.prototype._applyTransforms = function (data) {
 }
 
 Staticman.prototype._checkForSpam = function (fields) {
-  if (!this.config.akismet || !this.config.akismet.enabled) return Promise.resolve(fields)
+  if (!this.config.akismet || !this.config.akismet.type) return Promise.resolve(fields)
```

Comment:
No technical reason; I just thought it was a bit weird that this is the only property with a separate enable field. Other options are "enabled" when there's an non-empty field.